nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -100,17 +101,15 @@ def mean_minmax_statistic_collector(use_per_sample_stats: bool,
                                                 window_size)
 
     @staticmethod
-    def get_initializer_value(model: onnx.ModelProto, initializer_name: str) -> np.ndarray:
-        for initializer in model.graph.initializer:
-            if initializer.name == initializer_name:
-                return onnx.numpy_helper.to_array(initializer)
-        raise RuntimeError(
-            'There is no initializer with the name {}'.format(initializer_name))
+    def get_weight_tensor(model: onnx.ModelProto, node: NNCFNode) -> Tuple[str, np.ndarray]:
+        onnx_graph = ONNXGraph(model)
```

Comment:
Doesn't it have a performance impact, creating a graph from the model every time when weight tensor is needed?

Reply:
During the initialization of ONNXGraph there is no any computational operation is called.
But, we plan to think about refactoring of ONNXGraph and, probably, it will be completely removed and replaced by the simple functions which aim to get info from ONNX model.